Lecture3/Task1/utils/IOFileReader.java

```diff
@@ -0,0 +1,36 @@
+package utils;
+
+import java.io.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class IOFileReader {
+
+	public static void getFileInfo() {
```

Comment:
method with name `getFileInfo ` - shouldn't  be void

Reply:
return Productinfo

Reply:
fixed